src/Serialize.jl

```diff
@@ -1,19 +1,20 @@
 """
 `savejets(filename, jets; format="px py pz E")`
 
-Saves the given `jets` into a file with the given `filename`. Each line contains information about a single jet and is formatted according to the `format` string which defaults to `"E px py pz"` but can also contain other values in any order: `"pt"` or `"kt"` for transverse momentum, `"phi"` for azimuth, `"eta"` for pseudorapidity, `"m"` for mass. It is strongly NOT recommended to put something other than values and (possibly custom) separators in the `format` string.
+Saves the given `jets` into a file with the given `filename`. Each line contains information about a single jet and is formatted according to the `format` string which defaults to `"px py pz E"` but can also contain other values in any order: `"pt2"` or `"kt"` for transverse momentum, `"phi"` for azimuth, `"eta"` or `"rapidity"` for pseudorapidity, `"m"` for mass. It is strongly NOT recommended to put something other than values and (possibly custom) separators in the `format` string.
 """
 function savejets(filename, jets; format="px py pz E")
     symbols = Dict(
         "E" => JetReconstruction.energy,
         "px" => JetReconstruction.px,
-        "pt" => JetReconstruction.pt,
-        "kt" => JetReconstruction.pt,
+        "pt2" => JetReconstruction.pt2,
+        "kt" => JetReconstruction.pt2,
         "py" => JetReconstruction.py,
         "pz" => JetReconstruction.pz,
         "phi" => JetReconstruction.phi,
         "m" => JetReconstruction.mass,
-        "eta" => JetReconstruction.eta
+        "eta" => JetReconstruction.rapidity,
```

Comment:
should eta be eta?

Reply:
well when I first wrote that my idea was that we demand from the user to define eta() for their type. right now, according to the README.md, only pt2(), phi(), rapidity(), px(), py(), pz(), energy()  are required

Reply:
and as far as I remember, eta is rapidity (I could be wrong here though)

Reply:
eta is pseudorapidity, the two are only strictly equal when the object has `mass = 0`

Reply:
doesn't the rapidity actually give the same thing in the code? I honestly don't remember these details already